src/pad_controller.rs

```diff
@@ -7,14 +7,13 @@ use glib::IsA;
 use gtk_sys;
 use PadActionEntry;
 use PadController;
+use std::ffi::CString;
 
 pub trait PadControllerExtManual: 'static {
-    #[cfg(any(feature = "v3_22", feature = "dox"))]
     fn set_action_entries(&self, entries: &[PadActionEntry]);
 }
 
 impl<O: IsA<PadController>> PadControllerExtManual for O {
-    #[cfg(any(feature = "v3_22", feature = "dox"))]
```

Comment:
IMHO this is needed:
`<method name="set_action_entries" c:identifier="gtk_pad_controller_set_action_entries" version="3.22">`

Reply:
The whole file is under this compilation condition, take a look at `src/lib.rs`. ;)

Reply:
Thanks, I missed that.

Reply:
You're welcome!